scalableminds/webknossos
# isosurfaces for volume tracings

This PR makes it possible to fetch isosurfaces from the backend in volume tracings. An isosurface can be loaded by using `ctrl + leftMouse`. This PR adds a button to the 3d viewport that makes it possible to reload the fetched isosurfaces whose annotation changed. 

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a volume tracing and do some volume annotation.
- Load the isosurface of your annotation by usning `ctrl + leftMouse` in one of the plane viewports. The isosurface should be loaded. Then change the annotation of that cell and use the reload button in the top right corner of the 3d viewport. The isosurface should be erased and the reloaded to the newest version.

### Issues:
- fixes #3681 

------
